### PR TITLE
Amend docs

### PR DESCRIPTION
**Pull request by raxkaynan**

### CODE REVIEW

File: vuepress/docs/.vuepress/dist/docs/404.html [hunk 1]
```diff
@@ -3,17 +3,17 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width,initial-scale=1">
-    <title>Getting Started Guide</title>
+    <title>VuePress</title>
```
Comment (joewhite101) [inline vuepress/docs/.vuepress/dist/docs/404.html:6]: This doesn't look right to me.

Comment (raxkaynan) [reply vuepress/docs/.vuepress/dist/docs/404.html:6]: @es-entando seemed to remove the title, which originally was "Getting Started Guide". Vuepress by default generates that title when left empty.

**Comment by nshaw** [comment]
@raxkaynan, we've since removed the dist directory to avoid controlling generated code. Could you please rebase and resubmit any changes that survive?  Thanks.

**Comment by nshaw** [comment]
Closing since this looks out-of-date but feel free to reopen or open a new one if I misunderstood.

**Pull request closed without merging**